corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -225,19 +223,17 @@ def _check_for_exceeded_rate_limits(ip, num, user):
             per_minute=2,
             per_second=1,
         )
-    ).get_rate_limits(),
-    scope_length=1,
+    ).get_rate_limits(scope),
 )
 
 global_two_factor_setup_rate_limiter = RateLimiter(
     feature_key='global_two_factor_setup_attempts',
-    get_rate_limits=lambda: get_dynamic_rate_definition(
+    get_rate_limits=lambda scope: get_dynamic_rate_definition(
         'global_two_factor_setup_attempts',
         default=RateDefinition(
             per_day=100,
         )
     ).get_rate_limits(),
```

Comment:
Yeah, exactly, it doesn't because scope is not a required argument, and this limiter is global, but passing it in will work fine. It will end up the same default arg either way so I can make that change. I am hoping to refactor this code some more to replace these lambdas with a more clearly defined interface so hopefully it is short term either way.